models/anderson_2007.cpp

```diff
@@ -2,18 +2,27 @@
 // from anderson et al 2007 - Vancomycin pharmacokinetics in preterm neonates and 
 // the prediction of adult clearance.
 [PARAM] @annotated
-CL    : 3.83 : Clearance (L/hr)
-V     : 39.4 : Volume (L) 
-WT    : 2.9  : Weight (kg)
-PMA   : 34.8 : post-menstral age (wk)
-SCR   : 0.9  : serum creatinine (mg/dL)
-VENT  : 0    : Ventillation status
-IONT  : 0    : ionotrope status
-Kage  : 0.00789 : scaling constant of age on renal function
-Fvent : 1.03 : Scaling factor applied for use of positive pressure artificial ventilation
-Fiont : 1.19 : Scaling factor applied for use of inotropes (use of dopamine)
-OCC   : 1     : Occasion
-  
+CL        : 3.83    : Clearance (L/h)
+V         : 39.4    : Volume (L) 
+WT        : 2.9     : Weight (kg)
```

Comment:
those were quite arbitrary, mrgsolve (as of now) requires some default value, so I had just been carrying forward the original values from one of the first models I created, to keep some consistency in the default pseudo-individual across models.

For adults I am more clear regarding a canonical standard (esp 70kg weight) - do you have in mind any reasonable neonatal standard values?